src/herder/HerderImpl.cpp

```diff
@@ -1280,7 +1288,7 @@ HerderImpl::triggerNextLedger(uint32_t ledgerSeqToTrigger,
                                   SOROBAN_PROTOCOL_VERSION))
     {
         txPhases.emplace_back(
-            mSorobanTransactionQueue.getTransactions(lcl.header));
+            mSorobanTransactionQueue->getTransactions(lcl.header));
```

Comment:
Do we need to check that queue is initialized (just in case)? Same below

Reply:
If we got to this point (nominating a tx set in v20), Soroban tx queue must exist. If it doesn't, then core is in invalid state. I added extra asserts in both places to ensure the queue is initialized.